app/routes/road-marking-concepts/edit.js

```diff
@@ -14,6 +14,7 @@ export default class RoadmarkingConceptsEditRoute extends Route {
     });
   }
   resetController(controller) {
+    controller.reset();
     const roadMarkingConcept = controller.model.roadMarkingConcept;
```

Comment:
This can be moved into that reset method in the controller now (same for the other routes). Unneeded but it could be a nice refactor.